common/SmokeTests/Samples/IotHubConnection/IotHubConnection.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <GenerateProgramFile>false</GenerateProgramFile>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.1.0-dev.*" />
+    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.4.3" />
```

Comment:
I wasn't sure whether I should centralize these or leave them isolated.   It may make sense to shift them over.... thoughts?

Reply:
I don't have a pretty solution aside from pinning the dependencies at this time. I think this is OK as is. It might also make sense to allow the version to float so we can validate against later GA'd packages without having to remember to update dependency versions listed here.